lib/timeline/component/item/Item.js

```diff
@@ -169,8 +169,13 @@ Item.prototype._repaintDragCenter = function () {
     });
 
     if (this.dom.box) {
-      this.dom.box.appendChild(dragCenter);
-    } 
+      if (this.dom.dragLeft) {
+        this.dom.box.insertBefore(dragCenter, this.dom.dragLeft);
+      }
+      else {
```

Comment:
@yotamberk I added the example and checked against develop branch. In the develop branch the `dragCenter` is only added on selection, exactly as it's done in this PR. When `itemsAlwaysDraggable` is true, it is still **not** added, but the dragging works.